app/workers/talk_admin_create_worker.rb

```diff
@@ -3,6 +3,8 @@ class TalkAdminCreateWorker
 
   def perform(project_id)
     Project.find(project_id).create_talk_admin(client)
+  rescue TalkApiClient::NoTalkHostError => e
+    Rails.logger.info e
```

Comment:
Can you log an error in honey badger as well just so we know this is happening outside of the logs? As per [here](https://github.com/zooniverse/Panoptes/blob/db3030d0c4276f0668fab93f61eb3e41ae6f8564/app/controllers/api/events_controller.rb#L60)